phoebe/backend/universe.py

```diff
@@ -1877,19 +1886,24 @@ def _populate_lc(self, dataset, ignore_effects=False, **kwargs):
                                      photon_weighted=intens_weighting=='photon')
 
 
+
             # Beaming/boosting
             if boosting_method == 'none' or ignore_effects:
                 boost_factors = 1.0
-            elif boosting_method == 'linear':
-                logger.debug("calling pb.bindex for boosting_method='linear'")
-                bindex = pb.bindex(Teff=self.mesh.teffs.for_computations,
-                                   logg=self.mesh.loggs.for_computations,
-                                   abun=self.mesh.abuns.for_computations,
-                                   mu=abs(self.mesh.mus_for_computations),
-                                   atm=atm,
-                                   photon_weighted=intens_weighting=='photon')
-
-                boost_factors = 1.0 + bindex * self.mesh.velocities.for_computations[:,2]/37241.94167601236
+            elif boosting_method == 'manual':
+                # bindex = kwargs.get('boosting_index', self.boosting_index)
+                # bindex = kwargs.get('boosting_index', self.boosting_index.get(dataset, None))
+                boost_factors = 1.0 + bindex * self.mesh.velocities.for_computations[:, 2] / 37241.94167601236
+            # elif boosting_method == 'linear':
+            #     logger.debug("calling pb.bindex for boosting_method='linear'")
+            #     bindex = pb.bindex(Teff=self.mesh.teffs.for_computations,
+            #                        logg=self.mesh.loggs.for_computations,
+            #                        abun=self.mesh.abuns.for_computations,
+            #                        mu=abs(self.mesh.mus_for_computations),
+            #                        atm=atm,
+            #                        photon_weighted=intens_weighting=='photon')
+            #
+            #     boost_factors = 1.0 + bindex * self.mesh.velocities.for_computations[:,2]/37241.94167601236
```

Comment:
Let's just get rid of this since it won't work anyways - it will always live in the git history if we need to revive it.

```suggestion
```